src/blocks/FadeTransition.tsx

```diff
@@ -0,0 +1,62 @@
+import useConnections from './hooks/connections'
+import useProps from './hooks/props'
+
+import { ArgTuple, ComposableProps } from './types'
+
+export const options = {
+  actions: {},
+  hooks: {
+    show: {
+      args: [['args', 'any'] as ArgTuple],
+      returns: 'void',
+    },
+    hide: {
+      args: [['args', 'any'] as ArgTuple],
+      returns: 'void',
+    },
+  },
+  props: {
+    time: 'string',
+  },
+  state: {},
+  type: 'FadeTransition',
+}
+
+const FadeTransition: React.FC<ComposableProps> = ({
+  children,
+  connectConfig,
+  context,
+  id,
+  props,
+  className,
+}) => {
+  const actions = {}
+  const [isVisible, setIsVisible] = React.useState(false)
+  const { time } = useProps(context, props)
+  const hooks = {
+    /* eslint-disable */
+    show(args: { [key: string]: any }) {
+      setIsVisible(true)
+    },
+    hide(args: { [key: string]: any }) {
+      setIsVisible(false)
```

Comment:
This doesn't fade the children out, it only removes them. Can you also make it fade out? My suggestion is you have two state variables. One to set visibility and one to add the className. There could be two animation classNames, one for fade-in and one for fade-out.

Reply:
fade-out could just reverse the fade-in.